CHANGES.rst

```diff
@@ -1179,6 +1181,9 @@ New Features
 
 - ``astropy.io.fits``
 
+  - Fix convenience functions (`getdata`, `getheader`, `append`, `update`) to
```

Comment:
also, please use double backticks in the changelog, otherwise they will at some point turn into sphinx warnings...